tutor/src/components/change-student-id.cjsx

```diff
@@ -0,0 +1,104 @@
+_ = require 'lodash'
+React = require 'react'
+BS = require 'react-bootstrap'
+
+LoadableItem = require './loadable-item'
+{ ChangeStudentIdForm } = require 'shared'
+{ TransitionActions, TransitionStore } = require '../flux/transition'
+{ CourseStore, CourseActions } = require '../flux/course'
+{ StudentIdStore, StudentIdActions } = require '../flux/student-id'
+
+ERROR_MAP = {
+  student_identifier_has_already_been_taken:
+    'The provided student ID has already been used in this course. ' +
+    'Please try again or contact your instructor.'
+  blank_student_identifer:
+    'The student ID field cannot be left blank. Please enter your student ID.'
+}
+
```

Comment:
I'd prefer the `ERROR_MAP` to live in the store.  Then it could fire the appropriate error based on server error response, and `blank_student_identifer` could be emitted during the set inside the store's action, rather than in the component